syntax = "proto3";

package test;

service TestingService {
  rpc TestModel (TestRequest) returns (TestResponse);
}

message TestRequest {
  bytes model = 1;
  repeated double x_test = 2;
  repeated double y_test = 3;
  repeated string dates_test = 4;
}

message TestResponse {
  double rmse = 1;
  bytes plot = 2;
}
